src/app/series/directives/series-podcast.component.ts

```diff
@@ -66,9 +84,9 @@ export class SeriesPodcastComponent implements OnDestroy, DoCheck {
   }
 
   createDistribution() {
-    let podcast = new DistributionModel(this.series.doc);
-    podcast.set('kind', 'podcast');
-    this.series.distributions.push(podcast);
+    let podcastDist = new DistributionModel({series: this.series.doc, template: this.podcastTemplate.doc});
```

Comment:
I guess there's kind of a race condition here between loading the `versionTemplates` and the user clicking "Create Podcast".  But I'm not worried about it.